src/thunderbots/software/ai/hl/stp/action/chip_action.cpp

```diff
@@ -55,19 +57,19 @@ void ChipAction::calculateNextIntent(IntentCoroutine::push_type& yield)
     //                   |        \ /
     //                   >         A       <
     //                                     |
-    //                                     | DIST_TO_FRONT_OF_ROBOT / 2
+    //                                     | 0 dist (Point A is where the ball is)
```

Comment:
I'm confused. The ball is drawn below, but this says "Point A is where the ball is". Also what does "0 dist" mean?

Reply:
Yea, I changed that to properly reflect the code. Seemed a bit more clear to indicate Point A is at the ball rather than redraw with them overlapping? Thought it would get too crowded. Open to suggestions.

Reply:
fixed the ASCII diagram

Reply:
I think having them overlap would be a bit more clear, would be pretty reasonsable to say "A" is the ball location.

Reply:
Fixed again